src/binary/non_blocking/raw_binary_reader.rs

```diff
@@ -1218,7 +1218,8 @@ impl<'a, A: AsRef<[u8]>> TxReader<'a, A> {
         if type_descriptor.is_nop() {
             if let Some(item) = self.consume_nop_padding(&mut type_descriptor)? {
                 // We may encounter the end of the file or container while reading NOP padding,
-                // in which case `item` will be RawStreamItem::Nothing.
+                // in which case `item` will be RawStreamItem::Nothing. At the top level, it is
```

Comment:
Can't suggest change outside of diff, but looks like the fn comment needs updating.